zboxcore/zboxutil/http.go

```diff
@@ -260,6 +261,31 @@ func NewRefsRequest(baseUrl, allocationID, path, offsetPath, updatedDate, offset
 	return req, nil
 }
 
+func NewRecentlyAddedRefsRequest(bUrl, allocID string, fromDate, offset int64, pageLimit int) (*http.Request, error) {
+	nUrl, err := url.Parse(bUrl)
+	if err != nil {
+		return nil, err
+	}
+	nUrl.Path += RECENT_REFS_ENDPOINT + allocID
+
+	params := url.Values{}
+	params.Add("pageLimit", strconv.Itoa(pageLimit))
+	params.Add("offset", strconv.FormatInt(offset, 10))
+	params.Add("from_date", strconv.FormatInt(fromDate, 10))
```

Comment:
Let's unify the param format, so `from-date` and `page-limit`, or just `limit` for the page size perhaps.